src/Payment.ts

```diff
@@ -317,32 +318,108 @@ export class Payment {
     )
   }
 
-  public async getTransferInformation(
-    txHash: string,
+  /**
+   * Get the transfer details responsible for the transfer event with given log index and block number
+   * @param blockHash Block hash of the transfer event
+   * @param logIndex Log index of the transfer event
+   * @param options Optional network decimals for formatting the transfer value
+   */
+  public async getTransferDetails(
+    blockHash: string,
```

Comment:
I think we misunderstood us, I thought here to have the same argument as in the other function. So one object that includes both. Otherwise this does not fulfill what I wanted, because the other function I dont care about.

Reply:
Otherwise it should be good to go.